example/example.py

```diff
@@ -61,9 +62,10 @@
 
 # Visualize the robot
 params = {"link_axis_scale": 0.5}
-# vis = optas.RobotVisualizer(robot, q=q_nominal, params={})  # nominal
-vis = optas.RobotVisualizer(robot, q=q_solution, params=params)  # solution
+vis = Visualizer()  # solution
 
 # Draw goal position and start visualizer
-vis.draw_sphere(0.05, rgba=[0, 1, 0, 0.5], position=p_goal.toarray().flatten().tolist())
+vis.sphere(0.05, rgb=[0, 1, 0], position=p_goal.toarray().flatten().tolist())
+vis.robot(robot, q=q_solution)
```

Comment:
Good to show robot links and the names, there are some interface parameters you can set. See below

https://github.com/cmower/optas/blob/86434ce8b0f47e1e010c3bfd81d2dd85e09bd1d8/optas/visualize.py#L970-L985